cluster/gce/manifests/kube-addon-manager.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         - all
     # When updating version also bump it in:
     # - test/kubemark/resources/manifests/kube-addon-manager.yaml
-    image: k8s.gcr.io/addon-manager/kube-addon-manager:v9.1.5
```

Comment:
Let's separate the version bump into a follow-up PR - the actual chance needs to be merged first so that I can push the new image.

Reply:
done